root@0d64d6e5704e:/usr/share/koha/bin# diff -Naur /usr/share/koha/bin/koha-functions.sh /usr/share/koha/bin/koha-functions.sh.new 
--- /usr/share/koha/bin/koha-functions.sh 2015-05-24 05:18:40.000000000 +0200
+++ /usr/share/koha/bin/koha-functions.sh.new 2015-06-07 22:17:20.066025144 +0200
@@ -100,6 +100,7 @@
     local instancename=$1
 
     if daemon --name="$instancename-koha-zebra" \
+            --pidfiles="/var/run/koha/$instancename/" \
             --user="$instancename-koha.$instancename-koha" \
             --running ; then
         return 0
@@ -113,6 +114,7 @@
     local instancename=$1
 
     if daemon --name="$instancename-koha-indexer" \
+            --pidfiles="/var/run/koha/$instancename/" \
             --user="$instancename-koha.$instancename-koha" \
             --running ; then
         return 0
